stdlib/multiprocessing/context.pyi

```diff
@@ -42,12 +44,10 @@ class BaseContext:
     @staticmethod
     def active_children() -> list[BaseProcess]: ...
     def cpu_count(self) -> int: ...
-    # TODO: change return to SyncManager once a stub exists in multiprocessing.managers
-    def Manager(self) -> Any: ...
-    # TODO: change return to Pipe once a stub exists in multiprocessing.connection
-    def Pipe(self, duplex: bool = ...) -> Any: ...
+    def Manager(self) -> SyncManager: ...
+    def Pipe(self, duplex: bool = ...) -> tuple[Connection, Connection]: ...
```

Comment:
This seems to return a Pipe: https://github.com/python/cpython/blob/68fb03249f3b17146db42b00a75718b823a2280c/Lib/multiprocessing/context.py#L63

Reply:
Ugh, thanks multiprocessing. Should've known to look another level :-)

Hmm, on Windows it returns a tuple of PipeConnection https://github.com/python/cpython/blob/68fb03249f3b17146db42b00a75718b823a2280c/Lib/multiprocessing/connection.py#L577
Maybe we annotate this (and multiprocessing.connection.Pipe) as returning `tuple[_ConnectionBase, _ConnectionBase]`